test/ufuzz.js

```diff
@@ -582,6 +582,14 @@ function _createExpression(recurmax, noComma, stmtDepth, canThrow) {
       case 30:
         return '(' + createObjectLiteral(recurmax, COMMA_OK, stmtDepth, canThrow) + '.' +
             SAFE_KEYS[rng(SAFE_KEYS.length)] + ") ";
+      case 31:
+        var name = VAR_NAMES[rng(VAR_NAMES.length)];
+        return '(' + name + ' && ' + name + '[' +
+            createExpression(recurmax, COMMA_OK, stmtDepth, canThrow) + "]) ";
+      case 32:
+        var name = VAR_NAMES[rng(VAR_NAMES.length)];
```

Comment:
Done in https://github.com/mishoo/UglifyJS2/pull/1783/commits/75caa76353656d73c343bf9bd9df550e39bccc1e